src/test/java/de/tum/in/www1/artemis/ExamIntegrationTest.java

```diff
@@ -54,14 +64,20 @@
     @Autowired
     TextExerciseRepository textExerciseRepository;
 
+    @SpyBean
+    ExamAccessService examAccessService;
+
     private List<User> users;
 
     private Course course;
 
+    private Exam exam1;
```

Comment:
Do we have a better naming for this to show what the purpose of this attribute is?

Reply:
I will rename `course` to `course1`. I think it is a good practice to number related objects equally (the idea is that the course of `exam1` is `course1`...).